ruuvi_examples/ruuvi_firmware/sdk_application_config.h

```diff
@@ -20,7 +22,10 @@
   #define CRC16_ENABLED   FDS_CRC_ENABLED  
   #define CRC32_ENABLED   FDS_CRC_ENABLED  
 #endif
-#define NRF_LOG_ENABLED 0  // Disable log printout by default to save space
+
+#ifndef NRF_LOG_ENABLED

```

Comment:
Checking for if NRF_LOG_ENABLED is enabled here might cause some hard to track down bugs. It's better to throw error on redefinition. Please remove

Reply:
I don't understand what you are saying.

Without the` ifndef ` here and only the `define NRF_LOG_ENABLED 0`
main.c cannot `#define NRF_LOG_ENABLED 1`

Please reconsider.

Reply:
Exactly, If some .c files have log enabled and some do not there might be issues with conditional compiling of Nordic SDK log module.

Reply:
I have removed the conditional setting of NRF_LOG_ENABLED as you requested.